CocoaLumberjack.podspec

```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |s|
   end
 
   s.subspec 'Core' do |ss|
-    ss.source_files = 'Classes/DD*.{h,m}'
+    ss.source_files = 'Classes/DD*.{h,m}','Classes/*.swift'
```

Comment:
You could just add it above as `ss.source_files = 'Classes/CocoaLumberjack.{h,m,swift}'`.